cmd/multi-cluster/manifests/hub-cluster/leaf-cluster-secret.yaml

```diff
@@ -0,0 +1,8 @@
+#kubectl create secret generic leaf-cluster-token --from-literal=token=$(kubectl get secret default-token-p6kj6 -o jsonpath={.data.token} | base64 -d) -o yaml --dry-run=client
+apiVersion: v1
+data:
+  token: ZXlKaGJHY2lPaUpTVXpJMU5pSXNJbXRwWkNJNklsVnNXbmRYZFc5cVpIWm5Oall6WldSSmJHYzNiMmx0UjB0TWNXaGhXazkxVFZaNllsSlRTM1pWUzAwaWZRLmV5SnBjM01pT2lKcmRXSmxjbTVsZEdWekwzTmxjblpwWTJWaFkyTnZkVzUwSWl3aWEzVmlaWEp1WlhSbGN5NXBieTl6WlhKMmFXTmxZV05qYjNWdWRDOXVZVzFsYzNCaFkyVWlPaUptYkhWNExYTjVjM1JsYlNJc0ltdDFZbVZ5Ym1WMFpYTXVhVzh2YzJWeWRtbGpaV0ZqWTI5MWJuUXZjMlZqY21WMExtNWhiV1VpT2lKM1pXRjJaUzFuYVhSdmNITXRkRzlyWlc0dGVEZzNheklpTENKcmRXSmxjbTVsZEdWekxtbHZMM05sY25acFkyVmhZMk52ZFc1MEwzTmxjblpwWTJVdFlXTmpiM1Z1ZEM1dVlXMWxJam9pZDJWaGRtVXRaMmwwYjNCeklpd2lhM1ZpWlhKdVpYUmxjeTVwYnk5elpYSjJhV05sWVdOamIzVnVkQzl6WlhKMmFXTmxMV0ZqWTI5MWJuUXVkV2xrSWpvaVkyTTJOMkV6T1RNdFl6YzVZeTAwWkRsakxXRmpZbU10Tm1ObU1ETTRPV1l3TUdNeklpd2ljM1ZpSWpvaWMzbHpkR1Z0T25ObGNuWnBZMlZoWTJOdmRXNTBPbVpzZFhndGMzbHpkR1Z0T25kbFlYWmxMV2RwZEc5d2N5SjkuZkFPZy00X2o4dHR2cGRaYjVSN0I4U21RQ3Y5ZXNXWXZvS2VTZ05CcUU1SS1qZ3NLYjl6NHo1QlJoYUF2VmcwWXNKSXhKSWJxSkQyYU1RTnh1b0Rubk5DdmpiNU4wWUpWTTREQkMtcFdMNHRjVGFjckJzU0UtS3cwelVTNUU5S3R0NWt4UlcyMjhqalk1bTRBTWc4Qmo5OEpaZTR1MXl4QXhsUFpVYUdPZ3MwU1E1ZGl4TnIzR3RmZEdnYXZpaUxad1hMQzlCWE9aSmxvQ0d1aG5BZm4zZk5LSkV6aExQWndqTGpId2FNZ3JoeFZKU19CWTV1d2MxWVk4U2VfaUhXd2x2NXRGVlpGVTdNQjBYbUxoUm5YVjRkMnRzNjZCcF82Umc1SDNvY0VMN3N3NDNKaDBlcTJaSGtPaEJZUjFra2JMaG5hLXJPU0NlYy1NUWJobmpYWUp3
```

Comment:
Is this supposed to be here?

Reply:
that's a secret from a local cluster with no inbound access that doesn't exist anymore, so, no problem.